packages/webpack/src/buildtime/diagnostics.js

```diff
@@ -10,17 +10,18 @@ module.exports = {
    * @param {number} verbosity
    * @param {function} cb
    */
-  run: (lvl, cb) => {
-    if (level >= lvl) {
+  run: (verbosity, cb) => {
+    if (level >= verbosity) {
       return cb()
     }
   },
   /**
    * @param {number} verbosity
    * @param {any} content
    */
-  rawDebug: (lvl, content) => {
-    if (level >= lvl) {
+  rawDebug: (verbosity, content) => {
+    if (level >= verbosity) {
+      // @ts-ignore - trust me, the method exists
```

Comment:
a better fix--though **not at all necessary**--would be to create a `node.d.ts` file and extend the `NodeJS.Process` interface with the definition for `_rawDebug` 😄

Reply:
I prefer not to, it's not something we need in code completion or leaking form our package later ;)